main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2024 Alexander Dybdahl Troelsen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.uwaterloo.flix.api.lsp.provider.completion
+
+import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.api.lsp.Index
+import ca.uwaterloo.flix.language.ast.TypedAst
+
+/**
+  * A collection of keyword completers, including
+  *
+  * - `Decl`: completer for declaraiton keywords
+  * - `Enum`: completer for declaration keywords
+  * - `Expr`: completer for expressoin keywords
+  * - `Other`: completer for miscellaneous keywords
+  *
+  */
+object KeywordCompleter {
+  private def toCompletion(entry: (String, String => String)): Completion = {
```

Comment:
Get rid of this and just create each KeywordCompletion directly.

Reply:
Yep. I assume you mean specifically `toCompletion`, though I also did forget to clean up the documentation